src/app/pipe/truncate/truncate.pipe.ts

```diff
@@ -5,12 +5,17 @@ import { Pipe, PipeTransform } from '@angular/core';
  *
  * This is currently used with the save filter feature of the filter fields component
  */
-@Pipe({ name: 'Truncate'})
+@Pipe({ name: 'truncate'})
```

Comment:
This will be a breaking change for consumers of this pipe, right?  Can we add support for both Truncate and truncate to help with that?